pkg/kubelet/dockertools/docker.go

```diff
@@ -46,6 +48,13 @@ const (
 	DockerPrefix          = "docker://"
 )
 
+const (
```

Comment:
FWIU Rocket will be doing a similar conversion internally. Docker exposes shares so we do the conversion ourselves. The API we'd like to expose the Kubelet is millicores.